projects/plugins/crm/js/ZeroBSCRM.admin.global.js

```diff
@@ -348,6 +348,10 @@ function zbscrm_JS_initMenuPopups() {
 			position: 'bottom center',
 			hoverable: true,
 			on: 'hover',
+			delay: {
+				show: 300,
+				hide: 800,
+			},
```

Comment:
❤️

Reply:
The delay seems a bit high here. Maybe this?

```suggestion
			delay: {
				show: 50,
				hide: 400,
			},
```